hr_attendance_modification_tracking/__manifest__.py

```diff
@@ -1,13 +1,14 @@
 # Copyright 2019 Creu Blanca
+# Copyright 2020 Landoo
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    'name': 'Hr Attendance Modification Tracking',
+    'name': 'HR Attendance Modification Tracking',
     'summary': """
         Attendance changes will now be registered in the chatter.""",
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.1.0',
```

Comment:
Optional, as it's being taken care of during merge

Reply:
I agree. Can we change it without starting again with all revision process?

Reply:
@alexey-pelykh this case, Can I merge this pr with nobump?